assets/src/components/routeVariantName.tsx

```diff
@@ -22,3 +22,43 @@ export const RouteVariantName = ({ vehicle }: { vehicle: VehicleOrGhost }) => {
     </div>
   )
 }
+
+export const RouteVariantName2 = ({
+  vehicle,
+  className,
+  ...props
+}: { vehicle: VehicleOrGhost } & ComponentPropsWithoutRef<"output">) => {
+  const route = useRoute(vehicle.routeId)
+
+  const { routeId, viaVariant, headsign } = vehicle
+  const viaVariantFormatted = viaVariant && viaVariant !== "_" ? viaVariant : ""
+
+  const isShuttle = isVehicle(vehicle) && vehicle.isShuttle
+  return (
+    <output
+      title="Route Variant Name"
+      className={"m-route-variant-name " + className}
+      {...props}
+    >
+      {isShuttle ? (
+        "Shuttle"
+      ) : (
+        <>
+          <output
+            title="Route & Variant"
+            className="m-route-variant-name__route-id"
+          >
+            {`${route?.name || routeId}_${viaVariantFormatted}`}
+          </output>
+          &nbsp;
+          <output
+            title="Route Headsign"
```

Comment:
**nitpick:** I think this could be shortened to just "headsign" (I think of headsign as something associated with a particular vehicle or trip, not route).

Reply:
fixed in a6320b2309dd91740959877f5481ec449e577528